### PR TITLE
refactor(dop): remove core-services dependency on gittar

### PR DESCRIPTION
**Pull request by littlejiancc**
#### What this PR does / why we need it:
remove core-services dependency on gittar

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/gantt?filter__urlQuery=eyJpdGVyYXRpb24iOlsxMDkwXSwibWVtYmVyIjpbIjEwMDEyNjEiXX0%3D&id=279508&iterationID=1090&pId=267102&type=TASK)

#### Specified Reviewers:

/assign @sfwn 

#### ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |    remove core-services dependency on gittar          |
| 🇨🇳 中文    |    解除core-services依赖gittar          |

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: modules/core-services/services/application/appliation.go [hunk 1]
```diff
@@ -425,22 +393,6 @@ func (a *Application) Delete(applicationID int64) (*model.Application, error) {
 		return nil, errors.Wrap(err, "failed to delete application")
 	}
 
-	// 判断应用下是否有runtime，无 runtime 时应用可删除
```
Comment (sfwn) [inline modules/core-services/services/application/appliation.go:428]: What if the user calls core-services' to delete app with runtime directly?

File: apistructs/gittar.go [hunk 2]
```diff
@@ -771,6 +771,10 @@ type CheckRunRequest struct {
 	Branch string `json:"branch"`
 }
 
+func MakeRepoPath(orgName, proName, appName string) string {
```
Comment (sfwn) [inline apistructs/gittar.go:774]: add a `gittarutil` pkg is better.

**Review by sfwn — CHANGES_REQUESTED** [review]

File: apistructs/gittar.go (hunk 2 above)
Comment (littlejiancc) [reply apistructs/gittar.go]: done

**Review by sfwn — DISMISSED** [review]

**Pull request merged**